### DISCUSSION TITLE
part-ii/2019-3/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#94] [Question comments]
Part II 2019 Algebraic Geometry: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2019-3/

**Comment by Kiolt** [comment]
<details>
<summary>First part.</summary>

For (i) I got 3 components $V_1=\mathbb{V}(x,y^2-z^3)$ and $V_2=\{(-1,0,0)\},V_3=\{(1,0,-1)\}$. I'm worried that this is wrong.

(iii) Points are smooth. For $V_1$ only $(0,0,0)$ is singular.

</details>

<details>
<summary>Second part.</summary>

I encoded the tuple $(1,\alpha,\ldots,\alpha^{n-1})$ as a matrix and took its determinant. Matrix entries are polys so this should work.

</details>

**Reply by eha1234** [reply]
I got the same thing :)
